prog/vm/nexus.rb

```diff
@@ -177,6 +191,10 @@ def allocate
   def start
     register_deadline(:wait, 10 * 60)
 
+    unless vm.nics.all? { |n| n.strand.label == "wait" }
+      nap 10
```

Comment:
Put a pin in this for later: we should probably do a `donate` variant here, even though nic strands are not children, rather than waiting for the stars (or nap loops) to align.

Reply:
Given @byucesoy is kinda working his way into the strand system, I'm going to weigh it into his consciousness, too.

Reply:
You mean something like donate to non-child strand? Sure that would be useful. I'll think about it.